publisher-sdk/src/main/java/com/criteo/publisher/model/CdbResponse.java

```diff
@@ -76,7 +81,16 @@ public static CdbResponse fromJson(@NonNull JSONObject json) {
       }
     }
 
-    return new CdbResponse(slots, timeToNextCall);
+    boolean consentGiven = false;
```

Comment:
By having a default value to `false` instead of `null`, you should be sure that backend will always returns the consent even in case of no bid.

I'm ok with that, I made a ticket to check and implement this on backend: EE-1401